src/NServiceBus.Transport.Msmq/NonDurable/ExpressAttribute.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace NServiceBus.Transport.Msmq
+{
+    using System;
+
+    /// <summary>
+    /// Attribute to indicate that the message should not be written to disk.
+    /// This will make the message vulnerable to server crashes or restarts.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Interface)]
+    public sealed class ExpressAttribute : Attribute
```

Comment:
I though this was removed in v8? 

See https://docs.particular.net/nservicebus/upgrades/7to8/#non-durable-messaging and https://particularsoftware.slack.com/archives/C8A66739A/p1602158814032800 ?

Reply:
hm interesting, why isn't MSMQ mentioned given it does support it?